src/patternfly/components/Breadcrumb/breadcrumb.scss

```
@@ -78,7 +79,7 @@
   padding-inline-start: var(--#{$breadcrumb}__link--PaddingInlineStart); // use a mutable value for alignment control
   padding-inline-end: var(--#{$breadcrumb}__link--PaddingInlineEnd); // use a mutable value for alignment control
   font-size: inherit;
-  font-weight: var(--#{$breadcrumb}__link--FontWeight);
+  font-weight: inherit;
```

Technically this would be breaking - if a user had set `--#{$breadcrumb}__link--FontWeight` previously to add a custom font-weight here, this change will remove their custom weight. You can change it to below, then add `--#{$breadcrumb}__link--FontWeight: initial;` in the var list at the top of this stylesheet and that will make the font-weight `inherit` but also allow a user's customization using that variable to continue to work.

```suggestion
  font-weight: var(--#{$breadcrumb}__link--FontWeight, inherit);
```

---

Yup makes sense.